CMakeLists.txt

```diff
@@ -50,36 +50,6 @@
 #  Note can't use generator expressions on the source files since it interfers
 #  with setting the source property to target CUDA.
     target_sources (kron
-                    INTERFACE
-                    kgemm_nn_batched.hpp
-                    kgemm_nn.hpp
-                    kgemm_nt_batched.hpp
-                    kgemm_nt.hpp
-                    kroncommon.hpp
-                    kronmult1_batched.hpp
-                    kronmult1_pbatched.hpp
-                    kronmult1_xbatched.hpp
-                    kronmult2_batched.hpp
-                    kronmult2_pbatched.hpp
-                    kronmult2_xbatched.hpp
-                    kronmult2.hpp
-                    kronmult1.hpp
-                    kronmult3_batched.hpp
-                    kronmult3_pbatched.hpp
-                    kronmult3_xbatched.hpp
-                    kronmult3.hpp
-                    kronmult4_batched.hpp
-                    kronmult4_pbatched.hpp
-                    kronmult4_xbatched.hpp
-                    kronmult4.hpp
-                    kronmult5_batched.hpp
-                    kronmult5_pbatched.hpp
-                    kronmult5_xbatched.hpp
-                    kronmult5.hpp
-                    kronmult6_batched.hpp
-                    kronmult6_pbatched.hpp
-                    kronmult6_xbatched.hpp
-                    kronmult6.hpp
```

Comment:
Is anyone using generators that require a list of header files? If it's only visual studio and xcode, I agree we can remove this list.